collectors/python.d.plugin/postgres/postgres.chart.py

```diff
@@ -495,12 +504,13 @@
         ELSE pg_current_wal_lsn()
       END,
     replay_lsn) AS replay_delta
-FROM pg_stat_replication
+FROM pg_stat_replication psr
+LEFT OUTER JOIN pg_replication_slots prs on psr.pid = prs.active_pid
 WHERE application_name IS NOT NULL;
 """,
     V96: """
 SELECT
-    application_name,
+    coalesce(prs.slot_name, psr.application_name) application_name,
```

Comment:
```suggestion
    COALESCE(prs.slot_name, psr.application_name) application_name,
```

Let's make the function uppercase